package.json

```diff
@@ -20,7 +20,7 @@
     "> 1%"
   ],
   "dependencies": {
-    "@automattic/muriel-base-components": "file:packages/muriel-base-components",
+    "@automattic/tree-select": "^0.0.1",
```

Comment:
Is this intended to satisfy whatever is in the local `package.json`? Should we just use `*`?

> * := >=0.0.0 (Any version satisfies)
https://www.npmjs.com/package/semver#x-ranges-12x-1x-12-

Reply:
Since we don't publish wp-calypso, I think we could use *. In general you want to use whatever you'd see out in the wild, assuming things were published to npm.

Reply:
An advantage of `file:` is that you don't need to keep versions matching?

Hat `^` can be a problem when the package major version changes:

> include everything greater than a particular version in the same major range
> use the carat (aka hat) symbol, `^`
> examples:
> `^2.2.1`

https://semver.npmjs.com/

What's the motivation for using what you'd see in the wild? From what I understand, `*` or `>0.0.1` would be safer to ensure we're using the version in the local monorepo assuming that's the latest.